cmd/util/misc.go

```diff
@@ -94,3 +94,26 @@ func GetKVSecret(kvName, kvSecretName string) (secret string, err error) {
 
 	return *secretResp.Value, nil
 }
+
+func GetBlockSize(filesize int64, minBlockSize int64) (blockSize int64) {
+	blockSizeThreshold := int64(256 * 1024 * 1024) /* 256 MB */
+	blockSize = minBlockSize
+
+	/* We should not perform checks on filesize, block size limitation here. Those are performed in SDK
+	 * and take care of themselves when limits change
+	 */
+
+	for ; uint32(filesize/blockSize) > azblob.BlockBlobMaxBlocks; blockSize = 2 * blockSize {
+		if blockSize > blockSizeThreshold {
+			/*
+			 * For a RAM usage of 0.5G/core, we would have 4G memory on typical 8 core device, meaning at a blockSize of 256M,
```

Comment:
this comment doesn't seem to apply to Lustre, right?

Reply:
still applies, the machine is still limited by its memory.